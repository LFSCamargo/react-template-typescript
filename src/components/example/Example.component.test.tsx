import React from 'react';
import { shallow } from 'enzyme';

import Example from './Example.component';

describe('<Example />', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Example />);

    expect(wrapper).toMatchSnapshot();
  });
});
