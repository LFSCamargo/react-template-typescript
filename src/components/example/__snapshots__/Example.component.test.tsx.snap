// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Example /> should match snapshot 1`] = `
<Fragment>
  Example
  <br />
  <Styled(Button)
    loading={false}
    onClick={[Function]}
    type="primary"
  >
    Click Me
  </Styled(Button)>
</Fragment>
`;
